examples/with-one-login-per-subdomain/api-server.js

```diff
@@ -8,10 +8,7 @@ const apiPort = process.env.REACT_APP_API_PORT || 3001;
 const apiDomain = process.env.REACT_APP_API_URL || `http://example.com:${apiPort}`;
 const websitePort = process.env.REACT_APP_WEBSITE_PORT || 3000;
 const websiteDomain = process.env.REACT_APP_WEBSITE_URL || `http://example.com:${websitePort}`
-let whitelist = []
-whitelist.push(websiteDomain)
-whitelist.push(`http://a.example.com:${websitePort}`)
-whitelist.push(`http://b.example.com:${websitePort}`)
+let whitelist = /http?:\/\/([a-z0-9]+[.])example[.]com/
```

Comment:
This will also match `https://attacker.com/http://a.example.com` and would result in bypassing CORS. `/$http?:\/\/([a-z0-9]+[.])example[.]com/` would solve this partially at least.

Reply:
Alright, i'll update the whitelist with the above session

Reply:
Updated the regex, the previous regex we discussed was failing as the origins also have the port, Using the solution mentioned [here](https://stackoverflow.com/questions/14003332/access-control-allow-origin-wildcard-subdomains-ports-and-protocols)